### DISCUSSION TITLE
Just reinstalled today and poor oobabooga says "server.py: error: unrecognized arguments: --cai-chat"

### DISCUSSION THREAD

**Discussion by caterpillarpants** [oobabooga/text-generation-webui#1956] [Q&A] [answered]
Just reinstalled today and when I tried to run the start-webui script it says:

/oobabooga$ bash start-webui.sh
<code>INFO:Note: NumExpr detected 12 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
INFO:NumExpr defaulting to 8 threads.
INFO:Gradio HTTP request redirected to localhost :)
usage: server.py [-h] [--notebook] [--chat] [--character CHARACTER]
                 [--model MODEL] [--lora LORA [LORA ...]]
                 [--model-dir MODEL_DIR] [--lora-dir LORA_DIR] [--model-menu]
                 [--no-stream] [--settings SETTINGS]
                 [--extensions EXTENSIONS [EXTENSIONS ...]] [--verbose]
                 [--cpu] [--auto-devices]
                 [--gpu-memory GPU_MEMORY [GPU_MEMORY ...]]
                 [--cpu-memory CPU_MEMORY] [--disk]
                 [--disk-cache-dir DISK_CACHE_DIR] [--load-in-8bit] [--bf16]
                 [--no-cache] [--xformers] [--sdp-attention]
                 [--trust-remote-code] [--threads THREADS] [--n_batch N_BATCH]
                 [--no-mmap] [--mlock] [--wbits WBITS]
                 [--model_type MODEL_TYPE] [--groupsize GROUPSIZE]
                 [--pre_layer PRE_LAYER] [--checkpoint CHECKPOINT]
                 [--monkey-patch] [--quant_attn] [--warmup_autotune]
                 [--fused_mlp] [--flexgen] [--percent PERCENT [PERCENT ...]]
                 [--compress-weight] [--pin-weight [PIN_WEIGHT]] [--deepspeed]
                 [--nvme-offload-dir NVME_OFFLOAD_DIR]
                 [--local_rank LOCAL_RANK] [--rwkv-strategy RWKV_STRATEGY]
                 [--rwkv-cuda-on] [--listen] [--listen-host LISTEN_HOST]
                 [--listen-port LISTEN_PORT] [--share] [--auto-launch]
                 [--gradio-auth-path GRADIO_AUTH_PATH] [--api] [--public-api]
                 [--multimodal-pipeline MULTIMODAL_PIPELINE]
server.py: error: unrecognized arguments: `--cai-chat</code>

perhaps the script has a syntax error?
And how many threads do I really have with my Nvidia RTX 2070?

And thanks to the creators, maintainers and users of oobabooga. This is sooooo much fun. You rock!

**Comment by viperwasp** [comment, marked as answer]
This may not be it but I think something is off with "--cai-chat"
I suggest you change it to "--chat" in your arguments.

**Comment by caterpillarpants** [comment]
Thanks, viperwasp! It worked!

SOLVED!

**Comment by Hidrargiros** [comment]
Hello, I'm new to the community and I've been experiencing the same issue when trying to start the program through Google Colab Doc. I'm new to this and I don't know how to modify the code to make the "--cai-chat" error disappear. I've installed the model mayaeary/pygmalion-6b_dev-4bit-128g and I have an AMD on Windows, so I'm not sure if that affects anything when installing it online. Could you please help me by explaining how to solve the problem?
